x-pack/test_serverless/functional/test_suites/common/index.ts

```diff
@@ -13,6 +13,7 @@ export default function ({ loadTestFile }: FtrProviderContext) {
     loadTestFile(require.resolve('./management'));
 
     // platform security
+    loadTestFile(require.resolve('./security/api_keys'));
```

Comment:
This test suite was added as part of #164886 but was not referenced correctly so never got executed.

Reply:
Good catch!